.github/workflows/label_pull_request_onReview.yml

```diff
@@ -0,0 +1,51 @@
+# Workflow that labels a pr with the correct label
+name: "[Github Management] Pull Request Labeler on Review"
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  changes_requested:
+    if: github.event.review.state == 'changes_requested'
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Remove FE-Needs Review
+        uses: actions/github-script@v7
+        with:
+          script: |
+            github.rest.issues.removeLabel({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              name: ["FE-Needs Review"]
+            })
+      
+      - name: Remove FE-With QA
+        continue-on-error: true
+        uses: actions/github-script@v7
+        with:
+          script: |
+            github.rest.issues.removeLabel({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              name: ["FE-With QA"]
+            })
+
+      - name: Add the changes requested label
+        continue-on-error: true
+        uses: actions/github-script@v7
+        with:
+          script: |
+            github.rest.issues.addLabels({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            labels: ["FE-Changes Requested"]
+            })
```

Comment:
This can all be done in one step with the GH CLI:
```bash
gh pr edit ${{ github.event.pull_request.number }}  --remove-label "FE-Needs Review,FE-With QA" --add-label "FE-Changes Requested"
```

Just make sure to set the `GH_TOKEN` environment variable

Reply:
updated to the GH CLI